codalab/server/bundle_manager.py

```diff
@@ -659,15 +659,25 @@ def _get_staged_bundles_to_run(self):
         """
         # Keep track of staged bundles that have valid resources requested
         staged_bundles_to_run = []
+        # A dictionary structured as {owner_id: user information} to track those visited user information
```

Comment:
owner is a property of a bundle...this dictionary doesn't mention bundles, so just say `user_id`